File: SoftGallery/SoftGallery.Dominio/DTO/ProdutoDTO.cs
namespace SoftGallery.Dominio.DTO
{
    public class ProdutoDTO
    {
        public string nome { get; set; }
        public string? descricao { get; set; }
        public string? imagemUrl { get; set; }
        public decimal preco { get; set; }
    }
}

File: SoftGallery.Web/src/dtos/produtoDTO.ts
export interface ResumoProdutoDTO {
  id: string;
  nome: string;
  descricao?: string;
  imagemUrl?: string;
  preco: number;
  // Campos fictícios ou adicionados para simular front:
  tipo?: 'novidade' | 'destaque';
  avalicao?: number;
}
